p2p/backend-test-suite/src/block_announcement.rs

```diff
@@ -77,25 +75,19 @@ where
     connect_services::<S>(&mut conn1, &mut conn2).await;
 
     // Spam the message until until we have a peer.
-    loop {
-        let res = sync1
-            .make_announcement(Announcement::Block(
-                Block::new(
-                    vec![],
-                    Id::new(H256([0x01; 32])),
-                    BlockTimestamp::from_int_seconds(1337u64),
-                    ConsensusData::None,
-                    BlockReward::new(Vec::new()),
-                )
-                .unwrap(),
-            ))
-            .await;
-
-        match res {
-            Ok(()) => break,
-            Err(e) => assert_eq!(e, P2pError::PublishError(PublishError::InsufficientPeers)),
-        }
-    }
+    let res = sync1
```

Comment:
Should the comment be removed as there is no longer a loop?

Reply:
Removed, thanks.